### DISCUSSION TITLE
How to add header in traces

### DISCUSSION THREAD

**Discussion by raagulrakesh** [open-telemetry/opentelemetry-java-instrumentation#11063] [Q&A]
Hi,

I am using opentelemetry to traces in java code. I getting some much traces but I am not able to filter by headers in grafana.

**Comment by laurit** [comment]
cc @zeitlinger

**Comment by zeitlinger** [comment]
Assuming you mean HTTP headers - HTTP headers are not added as attributes spans automatically.

You can capture HTTP headers selectively: https://opentelemetry.io/docs/languages/java/automatic/configuration/#capturing-http-request-and-response-headers

**Reply by raagulrakesh** [reply]
Yes. I have added capture header it capturing header and pushing to tempo. but it pushing data as array so that not able to filter in grafana. is there is any option to convert array to string ?
